packages/deprecated/src/index.js

```diff
@@ -18,6 +19,13 @@ export const logged = Object.create( null );
  * @param {?string} options.hint        Additional message to help transition away from the deprecated feature.
  */
 export default function deprecated( feature, { version, alternative, plugin, link, hint } = {} ) {
```

Comment:
I think this could be shortened to:
```js
export default function deprecated( feature, options = {} ) {
const {
	version,
	alternative,
	plugin,
	link,
	hint,
} = options;
```